Resources/Prototypes/Entities/Structures/Doors/Airlocks/airlocks.yml

```
@@ -42,7 +42,7 @@
   suffix: Salvage
   components:
   - type: Sprite
-    sprite: Structures/Doors/Airlocks/Standard/salvage.rsi
+    sprite: SS220/Structures/Doors/Airlocks/Standart/salvage.rsi #SS220 salvage_airlock
```

```suggestion
    sprite: SS220/Structures/Doors/Airlocks/Standard/salvage.rsi #SS220 salvage_airlock
```

Как раз таки через d пишется